nixos/tests/sway.nix

```diff
@@ -66,30 +80,28 @@ import ./make-test-python.nix ({ pkgs, lib, ...} :
     machine.wait_for_file("/run/user/1000/wayland-1")
     machine.wait_for_file("/tmp/sway-ipc.sock")
 
-    # Test XWayland:
+    # Test XWayland (foot does not support X):
     machine.succeed(
         "su - alice -c 'swaymsg exec WINIT_UNIX_BACKEND=x11 WAYLAND_DISPLAY=invalid alacritty'"
     )
     machine.wait_for_text("alice@machine")
     machine.send_chars("test-x11\n")
     machine.wait_for_file("/tmp/test-x11-exit-ok")
-    print(machine.succeed("cat /tmp/test-x11.out"))
-    machine.screenshot("alacritty_glinfo")
+    machine.copy_from_vm("/tmp/test-x11.out")
+    machine.screenshot("alacrittyglinfo")
```

Comment:
Was the removal of the underscore intentional (below you're using `foot_wayland_info`)?

Reply:
no, I was running into another bug (#153766), thought it had something to do with the underscore and never changed it back.